lib/tasks/backfill_idv_level.rake

```diff
@@ -0,0 +1,48 @@
+namespace :profiles do
+  desc 'Backfill the idv_level column.'
+
+  ##
+  # Usage:
+  #
+  # bundle exec rake profiles:backfill_idv_level
+  #
+  task backfill_idv_level: :environment do |_task, _args|
+    with_statement_timeout do
+      is_in_person = Profile.where(id: InPersonEnrollment.select(:profile_id))
+      is_not_in_person = Profile.where.not(id: InPersonEnrollment.select(:profile_id))
+      needs_idv_level = Profile.where(idv_level: nil)
+
+      # rubocop:disable Rails/SkipsModelValidations
+      count = Profile.and(is_in_person).and(needs_idv_level).
+        update_all(idv_level: :legacy_in_person)
```

Comment:
do we have a rough sense of size of these?

I wonder if `find_in_batches` would be useful to make sure we don't make too big a query at once?

```suggestion
      count = 0
      Profile.and(is_in_person).and(needs_idv_level).find_in_batches do |batch|
        Profile.where(id: batch.map(&:id).update_all(idv_level: :legacy_in_person)
        count += batch.size
      end
```